gamatrix-gog.py

```diff
@@ -309,10 +307,24 @@ def set_multiplayer_status(game_list, cache):
         "-v", "--version", action="store_true", help="print version and exit"
     )
 
-    args = parser.parse_args()
+    return parser.parse_args(argv)
+
+
+if __name__ == "__main__":
+    logging.basicConfig(level=logging.INFO)
+    log = logging.getLogger()
+
+    args = parse_cmdline(sys.argv[1:])
+
     if args.debug:
         log.setLevel(logging.DEBUG)
 
+    # in case we want to see our command line shenanigans in a vebose session:
+    log.debug("Command line arguments:")
+    log.debug(sys.argv)
+    log.debug("Arguments after parsing:")
+    log.debug(args)
```

Comment:
Nit, but mind putting these on 2 lines, e.g.:

`log.debug(f"Command line arguments: {sys.argv}")`

You can put a newline in there if you want to retain that in the output. Not a big deal if you want to keep these as is.

Reply:
Done.